src/libstd/sys_common/gnu/libbacktrace.rs

```diff
@@ -9,6 +9,7 @@
 // except according to those terms.
 
 use libc;
+use backtrace_sys::*;
```

Comment:
Could we avoid the glob import here? At least I personally prefer to see exactly what's getting imported (and judging by the removed code, it's not that much).